AdobeStockAssetApi/Api/Data/CategoryInterface.php

```diff
@@ -64,4 +64,12 @@ public function getExtensionAttributes(): CategoryExtensionInterface;
      * @return void
      */
     public function setExtensionAttributes(CategoryExtensionInterface $extensionAttributes): void;
+
+    /**
+     * Set object as new
+     *
+     * @param bool $true
+     * @return void
+     */
+    public function isObjectNew(bool $true): void;
```

Comment:
Let's not add `isObjectNew` methods to an interface as using resource models for save operations is a temporary solutions